docs/installation/instructions.rst

```diff
@@ -1,25 +1,30 @@
-Installation Instructions
-=========================
+Pulp 3 Installation Instructions
+================================
 
-Pulp 3 Ansible Galaxy Installation
-----------------------------------
+.. note::
+
+    As Pulp 3 currently does not have an SELinux Policy, it currently requires the target
+    machine to have SELinux set to permissive mode::
+
+    $ sudo setenforce 0
```

Comment:
This is more of an FYI right, because the galaxy playbook and the vagrant env both disable selinux. The way its written now makes it seem like the user needs to take this step but in practice I don't think they will. I recommend moving this note further down so users don't think this is the first thing they need to do.

Reply:
Oh I see this is a step that users were being instructed to take. I guess it's good that people be required to turn it off manually. This is good as is, please disregard my comments.

Reply:
My thinking was that since it was currently listed as a step in the Ansible Galaxy instructions, it's nbd to just move it out and have it apply to all of them.

Reply:
Yeah I thought the galaxy playbook did this for you, but since it doesn't this is good.